### DISCUSSION TITLE
Mouse buttons not working properly while laser tablet drivers are active

### DISCUSSION THREAD

**Discussion by RadekMojzis** [ppy/osu#17571] [Q&A]
### Type

Game behaviour

### Bug description

While the in-game tablet driver is active, mouse buttons instantly recieve "button up" after they have been pressed (its impossible to drag/play sliders using mouse buttons)
Not sure how many people this affects, however I use a keypad where i have right/left mouse button bind for clicking in game - it makes it possible to navigate in menu/windows

in the video below i was mostly trying to drag - button is registered as down when cursor is pink
tryed it with open tablet drivers opend/closed, 
OS - W10 home (almost fresh install)
version - 21H2
build - 19044.1586

### Screenshots or videos

https://vimeo.com/manage/videos/691572681

### Version

2022.319.0-lazer

### Logs

[network.log](https://github.com/ppy/osu/files/8336707/network.log)
[performance.log](https://github.com/ppy/osu/files/8336703/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8336704/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8336705/updater.log)
[database.log](https://github.com/ppy/osu/files/8336706/database.log)

**Comment by bdach** [comment]
i cannot reproduce this, there is no information regarding your hardware, and the video cannot be viewed without creating an account.

**Comment by peppy** [comment]
Also sound like you have external software binding mouse buttons to keys. Please list any such software.

**Comment by RadekMojzis** [comment]
The keypad has software - Termite (now obsolete suceeded by alacrity) - it writes the input settings directly into the keypad itself.
Requires no drivers and works even on linux.
So there were no external programs running.
______________________________________________________________
> Od: "Dean Herbert" ***@***.***>
> Komu: "ppy/osu" ***@***.***>
> Datum: 24.03.2022 01:19
> Předmět: Re: [ppy/osu] Mouse buttons not working properly while laser tablet drivers are active (Issue #17423)
>
 
Also sound like you have external software binding mouse buttons to keys. Please list any such software.

**Comment by RadekMojzis** [comment]
Here i got a noahOverlay to show when key presses are registered outside of osu and used it as an overlay.
there are no other programs running except for osu, streamlabs and the noahboard
uploaded a new video to youtube link
https://youtu.be/Bzs0u-Wl9_I
______________________________________________________________
> Od: "Dean Herbert" ***@***.***>
> Komu: "ppy/osu" ***@***.***>
> Datum: 24.03.2022 01:19
> Předmět: Re: [ppy/osu] Mouse buttons not working properly while laser tablet drivers are active (Issue #17423)
>
 
Also sound like you have external software binding mouse buttons to keys. Please list any such software.

**Comment by peppy** [comment]
Going to migrate this to an discussion thread as there's very little we can do. Highly likely a configuration issue on the user's end.

**Comment by RadekMojzis** [comment]
Sorry for the nagging, i know this is not a priority right now.
I tested it on a different pc with just a mouse and a tablet connected - it is not related to the keypad in any way.

To reproduce the issue
1. activate tablet mode
2. click and hold with a mouse
3. move the cursor with the tablet.
As soon as the cursor is moved by the tablet, the click is released.
